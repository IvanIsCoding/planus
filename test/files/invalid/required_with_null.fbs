table Foo {
  x: string = null (required);
}