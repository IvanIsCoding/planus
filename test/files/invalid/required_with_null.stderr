[0m[1m[38;5;9merror[0m[1m: Cannot setup field as both required and optional[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/required_with_null.fbs:2:15
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: string = [0m[31mnull[0m (required);
  [0m[34mâ”‚[0m               [0m[31m^^^^[0m  [0m[34m--------[0m [0m[34mfield was declared required here[0m
  [0m[34mâ”‚[0m               [0m[31mâ”‚[0m      
  [0m[34mâ”‚[0m               [0m[31mfield was declared optional here[0m

Error: Bailing because of errors
